tests/baselines/albert_large_v2.json

```diff
@@ -8,6 +8,7 @@
                 "train_batch_size": 32,
                 "eval_f1": 92.2124,
                 "train_runtime": 3547.3268,
+                "train_samples_per_second": 1.0,
```

Comment:
is this real 1.0 for most of models? sound very low.

Reply:
No no I'm going to update it